core/search/src/main/java/org/exoplatform/services/wcm/search/base/LazyPageList.java

```diff
@@ -87,7 +85,7 @@ protected void populateCurrentPage(int page) throws Exception {
     currentListPage_ = new ArrayList<E>();
     if (queryData_.getQueryStatement() != null) {
       try {
-        SessionProvider provider = WCMCoreUtils.getSystemSessionProvider();
+        SessionProvider provider = WCMCoreUtils.getUserSessionProvider();
```

Comment:
Did you check if this class is used just for this case? olr is it used by other calsses for other documents paginating cases ? This change may cause regressions .